features/environment.py

```diff
@@ -528,6 +540,53 @@ def create_and_set_output_directory(self, feature_name):
         os.makedirs(feature_dir)
         self._output_dir = feature_dir
 
+    def clone(self, from_name, cluster_name, to_name):
+        f = self._processes[from_name]
+        custom_config = {
+            'scope': cluster_name,
+            'bootstrap': {
+                'method': 'pg_basebackup',
+                'pg_basebackup': {
+                    'command': self.BACKUP_SCRIPT + ' --walmethod=stream --dbname=' + f.backup_source
```

Comment:
How does the cloned node finds its data directory? It is started with PGDATA pointing to the backup destination (since there is no --datadir parameter to this script, perhaps, the PGDATA of the clone is used as a destination?)